docs/1.5/extensions/external-links.md

```diff
@@ -124,22 +124,40 @@ class MyCustomLinkRenderer implements InlineRendererInterface
 
 ## Adding Icons
 
-You can also use CSS to automagically add an external link icon by targeting the `html_class` given in the configuration:
+You can also use CSS to add a custom icon by targeting the `html_class` given in the configuration:
 
-```css
-// Font Awesome example:
+```php
+$config = [
+    'external_link' => [
+        'html_class' => 'external',
+    ],
+];
+```
+
+```scss
+// Font Awesome (https://fontawesome.com/icons/external-link-alt).
+a[target="_blank"]::after,
+a.external::after {
+   @extend .fa;       // Extend from font-awesome base styles.
+   content: "\f35d";  // fa-external-link-alt icon unicode.
+}
+
+// Bootstrap 3 Glyphicon (https://getbootstrap.com/docs/3.3/components/).
 a[target="_blank"]::after,
 a.external::after {
-   content: "\f08e";
-   font: normal normal normal 14px/1 FontAwesome;
+  @extend .glyphicon; // Extend from Glyphicon base styles.
+  content: "\e164";   // glyphicon-new-window icon unicode.
 }
 
-// Glyphicon example:
+// Custom SVG/Bootstrap Icons.
 a[target="_blank"]::after,
 a.external::after {
-  @extend .glyphicon;
-  content: "\e164";
-  margin-left: .5em;
-  margin-right: .25em;
+  display: inline-block;
+  content: "";
+  // Tip: use an SVG URL encoder (https://yoksel.github.io/url-encoder).
+  // https://icons.getbootstrap.com/icons/link-45deg/
+  background-image: url("data:image/svg+xml,%3Csvg class='bi bi-link-45deg' viewBox='0 0 16 16' fill='currentColor' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M4.715 6.542L3.343 7.914a3 3 0 104.243 4.243l1.828-1.829A3 3 0 008.586 5.5L8 6.086a1.001 1.001 0 00-.154.199 2 2 0 01.861 3.337L6.88 11.45a2 2 0 11-2.83-2.83l.793-.792a4.018 4.018 0 01-.128-1.287z'/%3E%3Cpath d='M5.712 6.96l.167-.167a1.99 1.99 0 01.896-.518 1.99 1.99 0 01.518-.896l.167-.167A3.004 3.004 0 006 5.499c-.22.46-.316.963-.288 1.46z'/%3E%3Cpath d='M6.586 4.672A3 3 0 007.414 9.5l.775-.776a2 2 0 01-.896-3.346L9.12 3.55a2 2 0 012.83 2.83l-.793.792c.112.42.155.855.128 1.287l1.372-1.372a3 3 0 00-4.243-4.243L6.586 4.672z'/%3E%3Cpath d='M10 9.5a2.99 2.99 0 00.288-1.46l-.167.167a1.99 1.99 0 01-.896.518 1.99 1.99 0 01-.518.896l-.167.167A3.004 3.004 0 0010 9.501z'/%3E%3C/svg%3E");
```

Comment:
Could we use the Octicon link SVG here, and put this example above the others?  Some users may want to stick with the same icon they're using from 1.4 and so having that specific example will help them migrate to 2.x a little easier.

And/or we could add something to the upgrade guide page  showing exactly how to keep the Octicon link in 1.5+.

Reply:
Let's just provide this one example using the Octicon link SVG in plain CSS. The main point of this is to show how to target and style via pseudo-selector. If someone chooses to use a different library, that's up to them.